--- faust-dist/compiler/Makefile.unix	2018-11-12 16:22:40.000000000 +0100
+++ faust-2.5.23/compiler/Makefile.unix	2018-11-12 16:28:24.000000000 +0100
@@ -156,5 +156,5 @@ else ifeq ($(LLVM_VERSION),$(filter $(LL
     CXXFLAGS += -std=gnu++11
 
-else ifeq ($(LLVM_VERSION),$(filter $(LLVM_VERSION), 6.0.0 6.0.0svn))
+else ifeq ($(LLVM_VERSION),$(filter $(LLVM_VERSION), 6.0.0 6.0.0svn 6.0.1))
     LLVM_VERSION = LLVM_60
     CLANGLIBS=$(CLANGLIBSLIST)
