--- a/x11docker	2023-01-18 08:46:57.000000000 -0800
+++ a/x11docker	2024-04-20 19:44:04.673784064 -0700
@@ -12,7 +12,7 @@
 # More documentation at:  https://github.com/mviereck/x11docker
 
 Version="7.6.0"
-Packagedversion="no" # Set to "yes" if you want to package x11docker. This disables installation options.
+Packagedversion="yes" # Set to "yes" if you want to package x11docker. This disables installation options.
 
 # --enforce-i: Enforce running in interactive mode to allow commands tty and weston-launch in special setups. (deprecated)
 grep -q -- "--enforce-i" <<< "$*" && case $- in
